### PR TITLE
Resize the rootfs while building the image

### PR DESCRIPTION
**Pull request by mkumatag**
Resize the rootfs on the host while building the image

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/27#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mkumatag** [comment]
/cc @bpradipt @sudeeshjohn 

This is an alternative solution to https://github.com/ocp-power-automation/infra/pull/24 for resizing the rootfs while building the image.

File: scripts/images/README.md
Comment (bpradipt) [inline scripts/images/README.md:19]: `cloud-utils-growpart` available in RHEL/CentOS 8.x ?

Comment (mkumatag) [reply scripts/images/README.md:19]: yes!
```shell
[root@ip9-114-181-59 mkumatag]# yum info cloud-utils-growpart
Updating Subscription Management repositories.
Last metadata expiration check: 2:48:02 ago on Thu 15 Oct 2020 11:05:47 PM EDT.
Installed Packages
Name         : cloud-utils-growpart
Version      : 0.29
Release      : 3.el8
Architecture : noarch
Size         : 61 k
Source       : cloud-utils-growpart-0.29-3.el8.src.rpm
Repository   : @System
From repo    : rhel-8-for-ppc64le-appstream-rpms
Summary      : Script for growing a partition
URL          : https://launchpad.net/cloud-utils
License      : GPLv3
Description  : This package provides the growpart script for growing a partition. It is
             : primarily used in cloud images in conjunction with the dracut-modules-growroot
             : package to grow the root partition on first boot.

[root@ip9-114-181-59 mkumatag]#
```

File: scripts/images/convert_qcow2_ova.py
Comment (bpradipt) [inline scripts/images/convert_qcow2_ova.py:328]: "resizing partition" ?

File: scripts/images/convert_qcow2_ova.py
Comment (bpradipt) [inline scripts/images/convert_qcow2_ova.py:335]: "resizing filesystem..." ?

File: scripts/images/convert_qcow2_ova.py
Comment (bpradipt) [inline scripts/images/convert_qcow2_ova.py:437]: print("Preparing ", imageDist, " image...") ??

File: scripts/images/convert_qcow2_ova.py
Comment (bpradipt) [inline scripts/images/convert_qcow2_ova.py:329]: nice..

File: scripts/images/convert_qcow2_ova.py
Comment (mkumatag) [reply scripts/images/convert_qcow2_ova.py:328]: will reword

File: scripts/images/convert_qcow2_ova.py
Comment (mkumatag) [reply scripts/images/convert_qcow2_ova.py:335]: will reword

File: scripts/images/convert_qcow2_ova.py
Comment (mkumatag) [reply scripts/images/convert_qcow2_ova.py:437]: It was just a copy-paste, but I will take care.

File: scripts/images/README.md
Comment (bpradipt) [reply scripts/images/README.md:19]: Thanks

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/27#pullrequestreview-510206043" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/27#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [bpradipt,mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
